### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by robinradx** [wpappointments/wpappointments#228] [General]
Hey there,

I am planing to implement your plugin in my project that is aimed to go public by the end of the year. Do you guys think there will be a stable version at this point? I already bought and after looking through the Hub I am even more stoked. Great work so far. But if the the schedule is off that much, I have to think about a work around for that time.

Best
Rob

**Comment by MGkBIZ** [comment]
there is information that there will be a demo version in 2 days, but very few functions.

**Comment by wppoland** [comment]
Thanks for following the roadmap. That demo note is from 2024 and is out of date. The project is active again and the focus right now is shipping v1.0 of the free core plugin, with integrations such as CalDAV and webhooks following after launch. Current progress is tracked in the repo issues and discussions.
